public/data/locales/en.json

```diff
@@ -10,5 +10,7 @@
     "forestfrontrunners": "Forest Frontrunners",
     "mostrecent": "MOST RECENT",
     "mosttrees": "MOST TREES",
-    "trees": "Trees"
+    "trees": "Trees",
+    "the": "The",
+    "forestGrows": "Forest Grows."
```

Comment:
This won't work in every language. And I am also not sure if capitalizing every word looks good in German, e.g. 'Der Klum Wald Wächst' - for me it looks unreadable not given the fact that is has a mistake and in correct grammar we would have to write "Klum-Wald".
We should use a i18n resource with a placeholder instead for flexibility for every language.

Reply:
Btw. we have not configured LingoHub for this project and need to add the German translation manually before merging!

Reply:
As adding i18n to this project would increase the project, we cannot use a placeholder solution. So I added (wrong) German translations as good as possible. now.